bld/namelist_files/namelist_definition.xml

```diff
@@ -2955,12 +2955,31 @@ Default: 10
 <!-- Moist Convection and Microphysics -->
 
 <entry id="deep_scheme" type="char*16" category="conv"
-       group="phys_ctl_nl" valid_values="ZM,UNICON,off,CLUBB_SGS" >
+       group="phys_ctl_nl" valid_values="ZM,UNICON,off,CLUBB_SGS,YOG" >
 Type of deep convection scheme employed.  'ZM' for Zhang-McFarlane;
-'off' for none; or 'UNICON' which doesn't distinquish shallow and deep.
+'off' for none;'YOG' for Yanni-O'Gorman; or 'UNICON' which doesn't
+distinquish shallow and deep.
 Default: 'ZM' unless using 'UNICON', 'SPCAM' or 'pbl=none'
```

Comment:
```suggestion
Default: 'ZM' unless using 'UNICON', 'SPCAM', 'YOG' or 'pbl=none'
```

Reply:
Good spot, agreed.

Reply:
This is outdated in the most recent refactor where YOG is specified separately from deep_scheme.